### PR TITLE
Feat/agenda 189

### PR DESCRIPTION
**Pull request by rajkallem66**
@duonggs  Could you please review

### CODE REVIEW

File: client/admin/viewmodels/request.js [hunk 1]
```diff
@@ -259,6 +259,25 @@ define(["plugins/http", "plugins/observable", "durandal/app", "eventHandler", "d
                     }
                 });
             };
+            this.displayAll = function(){
+                this.requests.forEach(function(request){
+                    if (request.status === "display" || request.status === "active") {
+                        request.status = "approved";
```
Comment (duonggs) [inline client/admin/viewmodels/request.js]: @rajkallem66 
Is line 265 needed? The very next line, you set the status to "display"

Comment (rajkallem66) [reply client/admin/viewmodels/request.js]: Thanks for catching @duonggs  .. We can remove  request.status = "approved"; 
There is an epic which would change this logic but for the stories on this PR this should work.

**Comment by gh0st** [comment]
I think it might be a good idea to merge this branch into develop, then into master since it's the most mature branch of them all. And then merge this develop into master and introduce a formal versioning system. Then master can be used for all production deployments. Develop can be used to branch off of fixing bugs, and implementing features, etc. And then remove all the stale branches. What do you think @rajkallem66, @duonggs?

**Comment by duonggs** [comment]
I agree. Except for the formal versioning, this is essentially how we do it with Atlassian/Bitbucket.

**Comment by gh0st** [comment]
@rajkallem66 if you're comfortable with this offAgenda branch and it's been deployed a few times or so, let's nuke ALL the other branches except this one. And then branch `develop` and `master` off this. Let's also install `standard-version` and start doing version management with it to keep track of what version rts is on.

**Comment by rajkallem66** [comment]
Yes i agree with you both . I would take off/agenda as our master and
remove other branches. I am in middle of developing a big feature for RTS
once that is done we can plan this effort. I have already created a task to
work on it .

**Pull request merged**